tests/unit/dataactvalidator/test_load_program_activity.py

```diff
@@ -1,19 +1,24 @@
 from io import StringIO
 from unittest.mock import patch
+import datetime
 
 from dataactvalidator.scripts import load_program_activity
-from dataactcore.models.domainModels import ProgramActivity
+from dataactcore.models.domainModels import ProgramActivity, ExternalDataType
 
 
-@patch('dataactvalidator.scripts.load_program_activity.boto')
-def test_get_program_activity_file_aws(boto, monkeypatch):
+@patch('dataactvalidator.scripts.load_program_activity.io.BytesIO')
+@patch('dataactvalidator.scripts.load_program_activity.boto3')
+def test_get_program_activity_file_aws(boto3, bytesio, monkeypatch):
 
-    monkeypatch.setattr(load_program_activity, 'CONFIG_BROKER', {'use_aws': True, 'aws_region': 'aws_region',
-                                                                 'sf_133_bucket': 'aws_bucket', })
+    monkeypatch.setattr(load_program_activity, 'CONFIG_BROKER', {'use_aws': True})
+
+    bytesio.return_value = None
 
     load_program_activity.get_program_activity_file('some path')
 
-    boto.s3.connect_to_region().lookup().get_key.assert_called_with('program_activity.csv')
+    boto3.resource('s3').Object(
+            load_program_activity.PA_BUCKET, load_program_activity.PA_SUB_KEY+load_program_activity.PA_FILE_NAME
+            ).get.assert_called_with(load_program_activity.PA_SUB_KEY+load_program_activity.PA_FILE_NAME)
 
 
 def test_get_program_activity_file_local(monkeypatch):
```

Comment:
This test is failing.

Reply:
This entire test file is passing locally. Looking into reproducing the failure.